### PR TITLE
Bug 1086687 - Write unlink facebook friends test

### PR DESCRIPTION
**Pull request by mwargers**

### CODE REVIEW

File: tests/python/gaia-ui-tests/gaiatest/tests/functional/contacts/test_unlink_facebook.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            self.testvars['facebook']
+        except KeyError:
+            raise SkipTest('Facebook account details not present in test variables')
+
+        self.contact = MockContact()
+        self.data_layer.insert_contact(self.contact)
+
+    def test_unlink_facebook_contact(self):
+        """
+
```
Comment (JohanLorenzo) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/contacts/test_unlink_facebook.py]: Extra blank line

File: tests/python/gaia-ui-tests/gaiatest/tests/functional/contacts/test_unlink_facebook.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        https://moztrap.mozilla.org/manage/case/5858/
+
```
Comment (JohanLorenzo) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/contacts/test_unlink_facebook.py]: Extra blank line

File: tests/python/gaia-ui-tests/gaiatest/apps/contacts/regions/contact_details.py [hunk 3]
```diff
@@ -58,6 +60,23 @@ def tap_send_sms(self):
...
 
+    def tap_link_contact(self):
+        facebook_button = Wait(self.marionette).until(expected.element_present(*self._facebook_link_locator))
+        Wait(self.marionette).until(expected.element_displayed(facebook_button))
+        facebook_button.tap()
+        from gaiatest.apps.system.regions.facebook import FacebookLogin
+        return FacebookLogin(self.marionette)
+
+    def tap_unlink_contact(self):
+        facebook_button = Wait(self.marionette).until(expected.element_present(*self._facebook_link_locator))
```
Comment (mwargers) [inline tests/python/gaia-ui-tests/gaiatest/apps/contacts/regions/contact_details.py]: Ok, but then I should rename the above facebook_button to facebook_link_button

**Comment by try-server-hook** [comment]
[mwargers Martijn (mwargers)](https://github.com/mwargers) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=7c8dff909463)

**Comment by chirarobert** [comment]
lgtm

**Pull request merged**